### DISCUSSION TITLE
local development CORS blocks request to postgres

### DISCUSSION THREAD

**Discussion by samuelstroschein** [supabase/supabase#3378] [Questions] [answered]
Hi, 

I am running a local supabase instance for development purposes. So far so good, but requests from the browser via `supabase.from('foo').select('*')` are being blocked by CORS.

Does anyone know how to set the CORS config for the supabase docker container(s) / how to solve the problem of CORS blocking requests from the browser? 

<img width="628" alt="Screenshot 2021-09-30 at 15 22 44" src="https://user-images.githubusercontent.com/35429197/135463506-be193a00-2546-49cb-b1c1-1296a932e761.png">

**Comment by soedirgo** [comment, marked as answer]
Those URLs look wrong - can you try the recently updated guide [here](https://supabase.io/docs/guides/self-hosting)?

**Reply by samuelstroschein** [reply]
Eh, wrong .env file. Solved :D
